scripts/init.mk

```diff
@@ -49,11 +49,20 @@ githooks-run: # Run git hooks configured in this repository @Operations
 		--all-files
 
 asdf-install: # Install asdf @Installation
-	if [ -d "${HOME}/.asdf" ]; then
-		( cd "${HOME}/.asdf"; git pull )
-	else
-		git clone --depth=1 https://github.com/asdf-vm/asdf.git "${HOME}/.asdf" ||:
+	asdf_version=$$(grep -E "#\s+asdf\s+v[0-9.]+\s+[a-fA-F0-9]{40}" .tool-versions | awk '{ print $$4 }')
```

Comment:
I can't right now see where this might check whether I've already got `asdf` installed.  Worth gating the whole thing behind `if ! command -v asdf; then...`?

Reply:
The make taget to install `asdf` is no longer present.